projects/crystal.md

```diff
@@ -0,0 +1,27 @@
+## Overview
+![ logo](../projects/logo/crystal.png)  
+Name: Crystal Clear Services    
+[Website](http://ccstoken.com/)  
+Year founded: 2015    
+Currency: CCT   
+ICO Date: 1 September 2017  
```

Comment:
ICO End date is even more important, because https://keycryptovc.slack.com/archives/C6BP9497A/p1503336474000295 
and 
https://keycryptovc.slack.com/archives/C6BP9497A/p1503409506000494